ui/static/ui/js/listing.js

```diff
@@ -355,12 +279,14 @@ define('listing',
         });
       });
 
-      $("a#repo_page_status").click(function() {
-        var pageNum = $(this).data('page-num');
-        var maxPages = $(this).data('max-page-num');
-        navigateToPage(pageNum, maxPages, this);
-      });
+      var updatePage = function(newPageNum) {
+        var urlMap = URI(window.location).query(true);
+        urlMap.page = newPageNum;
+        window.location = "?" + URI().search(urlMap).query();
+      };
```

Comment:
The `pageNum` is passed in the `loader` function above, we don't read it in the URL. This just sets the window location with the updated page parameter. In the near future we can update this callback to just change `pageNum` and call the API with the new page number.